Collections/Collections.pier

```diff
@@ -713,16 +713,17 @@ difference.
 
 A symbol is similar to a string but is guaranteed to be globally unique.  For
 this reason symbols are preferred to strings as keys for dictionaries, in
-particular for instances of ==IdentityDictionary==. See also Chapter *cha:basic*
-for more about ==String== and ==Symbol==.
+particular for instances of ==IdentityDictionary==. 
+% See also Chapter *cha:basic*
```

Comment:
You see it is better to have a broken ref like that we do not forget. But thanks for your time.